rockcraft.yaml

```diff
@@ -66,3 +66,21 @@ parts:
             dpkg-query -W -f "${dpkg_ops}\n" \
             --root=${CRAFT_PROJECT_DIR}/../parts/postgresql-snap/layer/ \
             >> $CRAFT_PRIME/usr/share/rocks/dpkg.query
+            # PostgreSQL extensions
+            pushd "${CRAFT_PRIME}/usr/share/postgresql/14/extension"
+                ln -f postgis-3.control postgis.control
+                ln -f address_standardizer-3.control \
+                      address_standardizer.control
+                ln -f address_standardizer_data_us-3.control \
+                      address_standardizer_data_us.control
+                ln -f postgis_raster-3.control postgis_raster.control
+                ln -f postgis_tiger_geocoder-3.control \
+                      postgis_tiger_geocoder.control
+                ln -f postgis_topology-3.control postgis_topology.control
+                ln -f postgis_topology-3.control postgis_topology.control
+            popd
+            # Required for postgis_raster and pltcl PostgreSQL extensions
+            export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${CRAFT_PRIME}/usr/lib/\
+                ${CRAFT_ARCH_TRIPLET}/blas:${CRAFT_PRIME}/usr/lib/\
+                ${CRAFT_ARCH_TRIPLET}/lapack"
+            export TCL_LIBRARY="${CRAFT_PRIME}/usr/share/tcltk/tcl8.6"
```

Comment:
```suggestion
```
This will not persist. The envvars will only be set during the execution of the `override-prime` script. Testing locally, it seems that the `overlay-packages` are sufficient for the charm.